gnss/gps_time.py

```diff
@@ -94,20 +100,21 @@ def gps_minus_utc_seconds(gpst):
       Returns the number (or an array of them) of leap second values.
     """
 
-    delta_utc = np.zeros(gpst.shape, np.int)
+    delta_utc = np.zeros(gpst.shape, int)
     delta_utc = np.array(delta_utc)
-    assert np.all(gpst >= np.datetime64('1999-01-01T00:00:13Z'))
+    assert np.all(gpst >= np.datetime64('1999-01-01T00:00:13'))
```

Comment:
What's up with this change?

Reply:
see https://github.com/swift-nav/pygnss/pull/8#discussion_r946235990